app/components/find/courses/about_schools_component/view.rb

```diff
@@ -21,11 +21,24 @@ def initialize(course)
 
         def render?
           published_how_school_placements_work.present? ||
-            program_type == 'higher_education_programme' ||
-            program_type == 'scitt_programme' ||
+            program_type.in?(%w[higher_education_programme scitt_programme]) ||
             study_sites.any? ||
             site_statuses.map(&:site).uniq.many? || preview?(params)
         end
+
+        def show_higher_education_guidance?
+          course.higher_education_programme? && view_config(:higher_education, :except_provider_codes).exclude?(course.provider.provider_code)
+        end
+
+        def show_scitt_guidance?
+          course.scitt_programme? && view_config(:scitt_programmes, :except_provider_codes).exclude?(course.provider_code)
+        end
+
+        private
+
+        def view_config(*path)
```

Comment:
Isn't the method's name a bit too generic? It does hardcode `course_information` and `where_you_will_train`. In the other `config_for` scenarios you are assigning it to `@course_information_config`.

Reply:
Yeah, needs more work.